bindings_wasm/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+// use std::collectioashMap;
```

Comment:
Maybe just remove this?

Reply:
gone. It may come back on Monday - actually implemented.